recipes/nr.cli/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "nr.cli" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1bf47100b8ca21b3f184ee6f27c34adcfb7007612f8f8aada570891ffed59bca"
+
+build:
+  number: 0
+  entry_points:
+    - nr = nr.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - nr
+  commands:
+    - nr --help
+
+about:
+  home: "https://github.com/NiklasRosenstein/python-nr.cli"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: ""
+  summary: "This package provides the nr command-line tool which then allows you to use any comman registered to the \"nr.cli.commands\" entrypoint."
```

Comment:
can you fix this summary pleaes?

Reply:
What's wrong about this summary? It used to result in a YAML syntax error from what `conda skeleton` spit out originally, where the double quotes hadn't been escaped. I added the backslashes to escape the double quotes.